### PR TITLE
LegacySettings

### PR DESCRIPTION
**Pull request by singhsarab**
Update runsetting with LegacySettings.

### CODE REVIEW

File: docs/configure.md [hunk 1]
```diff
@@ -152,6 +155,47 @@ document.
...
   </InProcDataCollectionRunSettings>
 
+  <!-- Adapter Configuration -->
+  <MSTest>
+    <!-- Enable legacy mode, legacy setting are honored only when this flag is set to true -->
+    <ForcedLegacyMode>true</ForcedLegacyMode>
+  </MSTest>
+
+  <!-- Settings used for Ordered tests and MSTest based tests when running in legacy mode -->
+  <LegacySettings>
```
Comment (kaadhina) [inline docs/configure.md]: please check if we want to have descriptive section similar to datacollector

File: docs/configure.md [hunk 2]
```diff
@@ -72,6 +73,8 @@ core principles for runsettings.
    test run via the test platform APIs. It may use it to read its own settings
    or take decisions (e.g. an adapter needs to disable appdomains if
    `DisableAppDomain` setting is provided via run settings).
+5. Users can now configure settings used for legacy scenarios like deployment,
+   scripts using run settings.
```
Comment (PBoraMSFT) [inline docs/configure.md]: Clarify that legacy scenarios are only applicable for MSTest v1 framework.

Comment (PBoraMSFT) [reply docs/configure.md]: Also add that this enables users to use all the settings that were available in testsettings using runsettings.

File: docs/configure.md [hunk 3]
```diff
@@ -347,6 +366,59 @@ a test run.
...
+Here is an example:
+```xml
+<Runsettings>
+  <MSTest>
+    <!-- Enable legacy mode, legacy setting are honored only when this flag is set to true -->
+    <ForcedLegacyMode>true</ForcedLegacyMode>
+  </MSTest>
+
+  <LegacySettings>
+    <Deployment>
```
Comment (PBoraMSFT) [inline docs/configure.md]: Check case for Deployment

File: docs/configure.md (hunk 2 above)
Comment (singhsarab) [reply docs/configure.md]: Covered these in the LegacySettings section below.

**Pull request merged**